test/e2e/network_create_test.go

```diff
@@ -58,7 +58,7 @@ func genericPluginsToPortMap(plugins interface{}, pluginType string) (network.Po
 func (p *PodmanTestIntegration) removeCNINetwork(name string) {
 	session := p.Podman([]string{"network", "rm", "-f", name})
 	session.WaitWithDefaultTimeout()
-	Expect(session.ExitCode()).To(BeZero())
+	Expect(session.ExitCode() < 2).To(BeTrue())
```

Comment:
I'm not a ginkgo expert nor fan, but would you consider `Expect(session.ExitCode()).To(BeNumerically("<=",1))` ? I believe that will give better diagnostics in case of error. (Those `false was not true` diagnostics always make me struggle to try to find out what happened).

Reply:
Sure, I was trying to discover the correct syntax for this.

Reply:
I just remembered seeing `Numerically` in the past, then grep'ed for it in `test/e2e`.